spec/models/subscription_spec.rb

```
@@ -64,19 +64,85 @@
+        end.to change { @subscription.reload.subscription_events.paused.count }.from(0).to(1)
+    
+        expect do
+          @subscription.update!(paused_at: Time.current)
+          expect(@subscription.reload.subscription_events.paused.last.occurred_at).to eq first_pause
+        end.not_to change { @subscription.reload.subscription_events.paused.count }
+      end
+
+      it "records a resumed event if paused_at is cleared and paused_at_previously_was is present" do
+        @subscription.update!(paused_at: 1.day.ago)
+        @subscription.reload
+    
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)